packages/eas-cli/src/credentials/ios/appstore/bundleIdCapabilities.ts

```diff
@@ -139,15 +93,98 @@ function shouldSkipPushNotificationsCapabilityUpdate(
     usesBroadcastPushNotifications: boolean;
   }
 ): boolean {
-  // For push notifications, we should always update the capabaility if
+  // For push notifications, we should always update the capability if
   // - settings are not defined in the existing capability, but usesBroadcastPushNotifications is enabled (we want to add settings for this capability)
   // - settings are defined in the existing capability, but usesBroadcastPushNotifications is disabled (we want to remove settings for this capability)
   const noSettingsAttributes = existing.attributes.settings == null;
   return !noSettingsAttributes === additionalOptions.usesBroadcastPushNotifications;
 }
 
+function shouldSkipIcloudCapabilityUpdate(
```

Comment:
this is removed in the next PR up the stack so not worth spending much time here